BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
```

Comment:
It would be useful to also have:

* How to run builds (the whole thing, or individual sections)
* The general flow of what happens (and what you get in /Distribution)